spec/checker_spec.lua

```diff
@@ -2,7 +2,8 @@ local driver = require 'pallene.driver'
 local util = require 'pallene.util'
 
 local function run_checker(code)
-    assert(util.set_file_contents("__test__.pln", code))
+    -- "__test__.pln" does not exist on disk. The compiler expects
+    -- a file with the ".pln" extension.
```

Comment:
This isn't actually true... The name is used in error messages but Pallene does not complain if we use a name that does not end in pln. What about "\_\_test\_\_.pln does not actually exist. The name is only used for error messages" ?

By the way, that is a weird place to put a line break. Usually we break after a sentence or at the 100th column.

Reply:
I am sure why I added a line break there. Will update.